.github/workflows/python-release.yml

```diff
@@ -17,16 +17,16 @@ jobs:
         with:
           python-version: '3.8'
 
-      - name: Regen openapi libs
-        run: |
-          yarn
-          ./regen_openapi.sh
-
       - name: Install setuptools
         run: |
           cd python
           python -m pip install setuptools
 
+      - name: Regen openapi libs
+        run: |
+          cd python
+          ./scripts/generate_openapi.sh
+
```

Comment:
sorry i didn't just mean setuptools, but you also now need the dev requirments installed since we now need to have `openapi-python-client` for this to work